docs/guides/common-tips.md

```diff
@@ -160,6 +174,104 @@ You can also update the props of an already-mounted component with the `wrapper.
 
 _For a full list of options, please see the [mount options section](../api/options.md) of the docs._
 
+### Mocking Transitions
+
+Although calling `await Vue.nexTick()` works well for most use cases, there are some situations where additional work arounds are required. These issues will be solved before the library transitions out of beta. One such example is unit testing components with the `<transition>` wrapper provided by Vue.
+
+```vue
+<template>
+  <div>
+    <transition>
+      <p v-if="show">Foo</p>
+    </transition>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      show: true
+    }
+  }
+}
+</script>
+```
+
+You might want to write a test that verifies that Foo is shown, then when `show` is set to `false`, Foo is no longer rendered. Such a test could be written as follows:
+
+```js
+test('should render Foo, then hide it', async () => {
+  const wrapper = mount(Foo)
+  expect(wrapper.text()).toMatch(/Foo/)
+
+  wrapper.setData({
+    show: false
+  })
+  await wrapper.vm.$nextTick()
+
+  expect(wrapper.text()).not.toMatch(/Foo/)
+})
+```
+
+In practice, although we are calling `setData` then waiting for the `nextTick` to ensure the DOM is updated, this test fails. This is an ongoing issue related to show Vue implements the `<transition`> component, that we would like to solve before version 1.0. For now, there are some work arounds:
+
+1. Using a `transitionStub` helper
```

Comment:
To structure this section and clearly show what belongs to it, I'd make this a heading.

```suggestion
#### Using a `transitionStub` helper
```